ncps/ncps.TESTGEN.ps1

```diff
@@ -0,0 +1,209 @@
+Param(
+    [Parameter(Mandatory=$false)] [String] $Config = "Default",

```

Comment:
All of these should be validated

Reply:
I validate in the main function so we can easily add configs without needing to change the TESTGEN script.